test/loading.jl

```diff
@@ -4,6 +4,8 @@ using Base.Test
 
 @test @__LINE__ == 5
 
+@test_throws ArgumentError include("test_sourcepath.jl\0")
```

Comment:
Maybe just call Core.include directly here? Otherwise, include_from_node1 is confusing the tests.